tests/sign.test.ts

```diff
@@ -165,3 +165,52 @@ test('Sign: custom signatureVerifyingKey', async (t) => {
 
   t.end()
 })
+// this test is here encase i want to stress test signing
+test.skip('Sign: 20 sign loop use me for "debuging"', async (t) => {
+  const run = promiseRunner(t)
+
+  /* Setup Network */
+  await run('network up', spinNetworkUp(NETWORK_TYPE))
+  t.teardown(async () => {
+    // this gets called after all tests are run
+    await eveEntropy.close()
+    await spinNetworkDown(NETWORK_TYPE).catch((error) =>
+      console.error('Error while spinning network down', error.message)
+    )
+  })
+
+  /* Setup Entropy */
+  await run('wasm', wasmGlobalsReady())
+  const eveEntropy = new Entropy({
+    keyring: new Keyring({ seed:eveSeed }),
+    endpoint: 'ws://127.0.0.1:9944',
+  })
+
+
+
+  await Promise.all([
+    await run('eveEntropy ready', eveEntropy.ready),
+  ])
+  await run('jump-start network', jumpStartNetwork(eveEntropy))
+
+  await run('register', eveEntropy.register())
+  let count = 0
+  const spl = []
+  while (count < 20) {
+      const msg = Buffer
+    .from('Hello world: new signature from eveEntropy!' + count)
+    .toString('hex')
+
+    spl.push(run(`sign loop ${count}`, eveEntropy.signWithAdaptersInOrder({
+      msg: { msg },
+      order: ['deviceKeyProxy'],
+    })))
+    ++count
+  }
+
+  await Promise.all(spl)
```

Comment:
maybe leave a comment here about what's expected here? or a simple test e.g.

```suggestion
  await Promise.all(spl)
    .then(() => t.pass('all signed'))
    .catch(err => {
      console.error(err)
      t.fail('all signed')
    })
```